tag_release.py

```diff
@@ -0,0 +1,554 @@
+#!/usr/bin/env python
+"""
+Tag repos for an Open edX release.
+"""
+from __future__ import unicode_literals, print_function
+import sys
+import re
+import json
+import getpass
+import argparse
+import logging
+try:
+    from path import Path as path
+    from git import Repo, Commit
+    from git.refs.symbolic import SymbolicReference
+    from git.exc import GitCommandError
+    import requests
+    from requests.exceptions import RequestException
+    import yaml
+except ImportError:
+    print("Error: missing dependencies! Please run this command to install them:")
+    print("pip install path.py requests GitPython PyYAML")
```

Comment:
This would be better handled with a requirements.txt file, even if named "tag_release_requirements.txt".